### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by def-** [nim-lang/nimble#119]
There should be a new release of nimble for the 0.11 compiler

**Comment by vladwing** [comment]
```nimble install nimble``` currently fails with:

```
Hint: version [Processing]
nimblepkg/version.nim(32, 30) Warning: use ValueError instead; EInvalidValue is deprecated [Deprecated]
nimblepkg/version.nim(58, 7) Warning: 'nil' statement is deprecated; use an empty 'discard' statement instead [NilStmt]
nimblepkg/version.nim(141, 22) Error: use '^' instead of '-'; negative indexing is obsolete
FAILURE: Execution failed with exit code 1
```

**Comment by dom96** [comment]
``nimble install nimble@#head`` will work.

**Comment by def-** [comment]
I don't think that should be the regular way of installing Nim packages for the stable compiler. Also, in some shells you have to use `nimble install "nimble@#head"`.

**Comment by dom96** [comment]
I agree. Perhaps it would be a good idea for Nimble to ask the user if they want to try GIT HEAD when the latest tagged version fails to compile.

**Comment by def-** [comment]
Why not release a new version of Nimble instead?

**Comment by dom96** [comment]
I can do that, but other packages have these problems too.

**Comment by dom96** [comment]
Will fix this via #139
